docs/relayer/guide.md

```diff
@@ -365,3 +365,34 @@ await polkaBTC.stakedRelayer.deregister();
 You can monitor the operation of your Relayer on the Relayer dashboard by adding the key to the [polkadot{.js} extension](https://polkadot.js.org/extension/).
 
 Once the Relayer is up and running, a "Relayer" tab will appear in the topbar of the app at [beta.polkabtc.io](https://beta.polkabtc.io/) (or you can access directly at [beta.polkabtc.io/relayer](https://beta.polkabtc.io/relayer)).
+
+## Upgrading
+
+We will announce on public channels when a new release is made available for the Relayer client. The changelog and binaries will be published on the [release page](https://github.com/interlay/polkabtc-clients/releases).
+
+### 1. Stop the Relayer client
+
+```
+# Quickstart
+docker-compose down staked_relayer
+
+# Standard Installation
+systemctl stop polkabtc-relayer.service
+
+# Standard / Source
+Ctrl+C
+```
+
+### 2. Download the script and/or binary
+
+Depending on the method of installation;
+
+* Quickstart:
+  1. Update the client version in `docker-compose.yaml`
```

Comment:
I think we use `wget` which normally makes a copy like `docker-compose.yaml.1`. Should we add an `rm` command so that they delete the old versions?

Reply:
I think it would be easier if they would just manually update the `docker-compose.yaml` directly, but happy to write the instructions to re-download the file.

Reply:
I think blind copy paste is normally the way people do things (at least that is true for me)